WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Downloadable Files/ProductDownloadSettingsViewModelTests.swift

```diff
@@ -0,0 +1,259 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class ProductDownloadSettingsViewModelTests: XCTestCase {
+
+    // MARK: - Initialization
+
+    func test_readonly_values_are_as_expected_after_initializing_download_settings() {
+        // Arrange
+        let product = MockProduct().product(downloadable: true)
+        let model = EditableProductModel(product: product)
+
+        // Act
+        let viewModel = ProductDownloadSettingsViewModel(product: model)
+
+        // Assert
+        XCTAssertEqual(viewModel.downloadLimit, 1)
+        XCTAssertEqual(viewModel.downloadExpiry, 1)
```

Comment:
nit: I was wondering where these values come from, maybe we can make them more explicit by setting them via `copy(downloadLimit:downloadExpiry:)` in L11 above?